src/lib/features.js

```diff
@@ -190,6 +190,23 @@ var featureInfos =
         },
         attributes: { 'web-worker': 'web-worker-restriction' },
     },
+    CALL_ON_GLOBAL:
+    {
+        description:
+        'The ability to call a function on the global object when invoking ' +
+        'Function.prototype.call without binding.',
+        check:
+        function ()
+        {
+            try
+            {
+                [].concat.call();
+                return true;
```

Comment:
In IE9 `[].concat.call()` produces an array with a single `undefined` element. So maybe like this?

```suggestion
                if ([].concat.call()[0])
                    return true;
```